items_test.go

```diff
@@ -0,0 +1,59 @@
+package fatturapa_test
```

Comment:
I added these tests to cover the preexisting functionality in `items.go`. I made sure the tests passed in main, before the rest of the changes in this PR.

Reply:
Top.